### DISCUSSION TITLE
Still no luck on Table metric

### DISCUSSION THREAD

**Discussion by daanadriaan** [laravel/nova-issues#4580] [Q&A] [answered]
- Laravel Version: 9
- Nova Version: 4.12.6
- PHP Version: 8

### Description:
Just tried out the new table metric of v4.12.6. I created the exact metric as in the documents, but it doesn't show up when I try to include in in my custom dashboard.

**Comment by juse-less** [comment, marked as answer]
Two things to always try, when things aren't shown in the browser is
1. Re-publish the relevant bundled assets (JavaScript and CSS)
    `php artisan vendor:publish --tag=nova-assets --force`
1. Then also clear caches (mostly just views that are related)
    `php artisan view:clear`, or `php artisan optimize:clear` for everything

This is usually something you'd want to do every time you update packages that has bundled assets.

Nova also has a section in [their documentation](https://nova.laravel.com/docs/4.0/installation.html#updating-nova-s-assets) to keep your assets up-to-date.
I believe it should have the same outcome as point (1) above, but will also publish language files.
